index.html

```diff
@@ -0,0 +1,861 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="main.css">
+  <title>The Next Web</title>
+</head>
+
+<body>
+  <nav>
```

Comment:
I like `position: sticky`, although is a relatively new property for CSS, it might have some browser compatibility issues, but it does the work asked by the previews reviewer so It is a ok.